schema/rdls_schema.json

```
@@ -866,9 +866,10 @@
           "exclusiveMinimum": 0
         },
         "coordinate_system": {
-          "title": "Reference coordinate system",
+          "title": "Coordinate reference system",
           "type": "string",
-          "description": "The reference co-ordinate system of this resource, using the [EPSG codelist](https://epsg.org).",
+          "description": "The co-ordinate reference system of this resource, using the EPSG or ESRI codes from https://epsg.io/. Provide both the CRS authority scheme and the code separated by a colon, e.g. EPSG:4326.",
```

https://epsg.io/ isn't an authoritative source, but we can reference it as a useful resource. The authoritative list of EPSG codes is available at https://epsg.org/. The only authoritative list of ESRI codes I can find is at https://github.com/Esri/projection-engine-db-doc, but I don't think that's a particularly helpful page to link to so I just linked the ESRI homepage instead.

```suggestion
          "description": "An identifier string for the coordinate reference system (CRS) used in this resource. The CRS must be identified by a code issued by either [EPSG](https://epsg.org/) or [ESRI](https://www.esri.com/). The identifier string format is {authority}:{code}, e.g. EPSG:4326. [https://epsg.io/](https://epsg.io/) provides a searchable database of EPSG and ESRI coordinate system codes."
```

---

Could use https://spatialreference.org/ref/esri/ as a more useful list of ESRI codes?

---

@stufraser1 the issue with that website is the same as for epsg.io, it isn't an authoritative source.